lib/utilities/default_epicboxes.dart

```diff
@@ -4,8 +4,8 @@ abstract class DefaultEpicBoxes {
   static const String defaultName = "Default";
 
   static List<EpicBoxServerModel> get all =>
-      [americas /*, asia, europe*/]; // Temporarily disable non-americas regions
-  static List<String> get defaultIds => ['americas' /*, 'asia', 'europe'*/];
+      [americas, asia, europe]; // Temporarily disable non-americas regions
```

Comment:
I think the asian and europe regional servers needs removing due to recently wanting only the americas server available